enterprise_access/settings/base.py

```diff
@@ -494,3 +494,9 @@ def root(*path_fragments):
 
 # Cache timeouts
 SUBSIDY_RECORD_CACHE_TIMEOUT = 60 * 5
+
+BRAZE_GROUPS_EMAIL_AUTO_REMINDER_DAY_5_CAMPAIGN = ''
```

Comment:
Reason for these blank values?

Reply:
these values are injected from `edx-internal` during runtime. see more info here in thread
https://twou.slack.com/archives/C049BVC75J7/p1711483552472069?thread_ts=1711482679.758729&cid=C049BVC75J7